app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(team: int) -> int:
+    return sum(player.get_rating() for player in team)
+
+
+def elves_concert(elves: str) -> None:
```

Comment:
The parameter `elves` is annotated as `str`, but it seems to be used as an iterable of objects with a `play_elf_song` method. Consider changing the annotation to a more appropriate type, such as `list` or `Iterable`.